red.py

```diff
@@ -488,8 +491,11 @@ def main():
     if settings.owner == "id_here":
         print("Owner has not been set yet. Do '{}set owner' in chat to set yourself as owner.".format(bot.command_prefix[0]))
     else:
-        owner.hidden = True # Hides the set owner command from help
-    yield from bot.login(settings.email, settings.password)
+        owner.hidden = True  # Hides the set owner command from help
+    if settings.logintype == "email":
```

Comment:
Promise, this ain't gonna work unless you make the necessary changes [here](https://github.com/Twentysix26/Red-DiscordBot/blob/develop/cogs/utils/settings.py)